docs/Microservices/RPC/commandsRPC.md

```diff
@@ -0,0 +1,87 @@
+
+RPC is primarily used for communicating with the Remote Provision Server (RPS) for activating and/or deactivating AMT devices. Additional optional arguments allow for things such as easier development testing or for use in network environments utilizing proxies.
+
+All currently available arguments and their definitions are listed below along with example commands. 
+
+### RPC Usage
+On Windows:
+``` bash
+rpc <required> [optional]
+rpc <informational>
+```
+
+On Linux:
+``` bash
+sudo ./rpc <required> [optional]
+./rpc <informational>
```

Comment:
May want to be explicit that this command must be invoked with sudo as well:

   sudo ./rpc  informational